public/app/plugins/panel/trend/TrendPanel.tsx

```diff
@@ -92,9 +92,7 @@ export const TrendPanel = ({
       options={options}
     >
       {(config, alignedDataFrame) => {
-        if (
-          alignedDataFrame.fields.filter((f) => f.config.links !== undefined && f.config.links.length > 0).length > 0
-        ) {
+        if (alignedDataFrame.fields.some((f) => Boolean(f.config.links?.length))) {
```

Comment:
unrelated, but same as https://github.com/grafana/grafana/commit/7d4f7e5ffcdc36d14f438885f0d49f3c962d1ed6#diff-cbf74ec9ee5bbed26d0d2cf4a37cc859637baf2c0e128655aa0ffe86a57e0638 (which this was copied from originally)